src/tv2_afvd_showstyle/getRundown.ts

```diff
@@ -542,18 +542,21 @@ function getGlobalAdLibPiecesAFVD(context: IStudioUserContext, config: Blueprint
 				path: 'BG_LOADER_SC',
 				ignoreMediaObjectStatus: true,
 				timelineObjects: _.compact<TSR.TSRTimelineObj>([
-					literal<TSR.TimelineObjVIZMSEElementInternal>({
-						id: '',
-						enable: { start: 0 },
-						priority: 110,
-						layer: GraphicLLayer.GraphicLLayerDesign,
-						content: {
-							deviceType: TSR.DeviceType.VIZMSE,
-							type: TSR.TimelineContentTypeVizMSE.ELEMENT_INTERNAL,
-							templateName: 'BG_LOADER_SC',
-							templateData: []
-						}
-					}),
+					config.studio.GraphicsType === 'VIZ'
+						? literal<TSR.TimelineObjVIZMSEElementInternal>({
+								id: '',
+								enable: { start: 0 },
+								priority: 110,
+								layer: GraphicLLayer.GraphicLLayerDesign,
+								content: {
+									deviceType: TSR.DeviceType.VIZMSE,
+									type: TSR.TimelineContentTypeVizMSE.ELEMENT_INTERNAL,
+									templateName: 'BG_LOADER_SC',
+									templateData: [],
+									showId: config.selectedGraphicsSetup.OvlShowId
+								}
+						  })
+						: null,
```

Comment:
Not a fan of deliberately adding null, it forces us to do null check further down the line. 
If we extract the array of timelineObjects to a local variable, then we can just push the timelineObject if the type is Viz and do nothing if it is not.